content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +81,193 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
+
+        export BOOST_ROOT=/usr/local/Cellar/boost/1.67.0_1
+
+{{n.next()}}. Source the file you added the `BOOST_ROOT` environment variable to. For example:
+
+        source .bashrc
+
+{{n.next()}}. Clone the `rippled` source code into your desired location and access the `rippled` directory.  ***TODO: Note from PM: Cloning github repo is going to require proper github setup. Should we provide some guidance or references? JHA: What is the proper github setup? I may have set this up a whlie ago and I can't remember the steps that may have been required.***
+
+        git clone git@github.com:ripple/rippled.git
+        cd rippled
+
+{{n.next()}}. By default, cloning puts you on the `develop` branch. Use this branch if you are doing development work and want the latest set of untested features.
+
+      If you want the latest stable release, checkout the `master` branch.
+
+        git checkout master
+
+      Or, you can checkout one of the tagged releases listed on [GitHub](https://github.com/ripple/rippled/releases).
+
+      If you want to test out the latest release candidate, checkout the `release` branch:
+
+        git checkout release
+
+{{n.next()}}. Create your `rippled` build directory and access it. For example:
+
+        mkdir my_build
+        cd my_build
+
+{{n.next()}}. Build `rippled`. This could take about 5 minutes, depending on your hardware specs.
+
+        cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug ..
+
+      You can set `CMAKE_BUILD_TYPE` to the `Debug` or `Release` build type. All four standard [`CMAKE_BUILD_TYPE`](https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html) values are supported.
+
+{{n.next()}}. Run the build using CMake. This could take about 10 minutes, depending on your hardware specs.
+
+        cmake --build . -- -j 4
+
+      This example uses a `-j` parameter set to `4`, which uses four processes to build in parallel. The best number of processes to use depends on how many CPU cores your hardware has available. Use `sysctl -a | grep machdep.cpu | grep core_count` to get your CPU core count. ***TODO: the mac build page in the github repo goes much more in-depth about cmake build options - should we include these here? https://github.com/ripple/rippled/tree/develop/Builds/macos#generate-and-build***
+
+{{n.next()}}. Run unit tests built into the server executable. This could take about 5 minutes, depending on your hardware specs. (optional, but recommended) ***TODO: We should provide info about what to do if you get failures. What should the user do?***
```

Comment:
Yes agreed, we need to learn what those failures are. I can ask.

Reply:
Fantastic - thank you!